package.json

```diff
@@ -27,7 +27,8 @@
     "@octokit/plugin-rest-endpoint-methods": "^7.1.1",
     "@octokit/plugin-retry": "^4.1.3",
     "@octokit/plugin-throttling": "^5.2.2",
-    "@octokit/types": "^9.2.2"
+    "@octokit/types": "^9.2.2",
+    "@octokit/request-error": "^4.0.2"
```

Comment:
Can you drop this down to 3.x, 4.x drops Node 14 and v16 support and we haven't done so in this repo yet

Reply:
@wolfy1339 done :)

Reply:
@wolfy1339 btw... with downgraded versions, plugin-retry and this (request error) have issue with declarations. With the latest one problem does not exists.

![image](https://github.com/octokit/octokit.js/assets/3431813/b273220d-150f-4dfe-91ec-bcbfabbaf5b8)

Reply:
That shouldn't be the case. It's most likely VSCode needing a refresh, as sometimes it likes to cache errors that are no longer there.

Browsing the npm package, you can see the types are there: https://unpkg.com/browse/@octokit/request-error@3.0.3/dist-types/index.d.ts